pkg/apis/serving/v1alpha1/revision_defaults.go

```diff
@@ -41,9 +41,9 @@ func (rs *RevisionSpec) SetDefaults(ctx context.Context) {
 	}
 
 	// When ConcurrencyModel is specified but ContainerConcurrency
-	// is not (0), use the ConcurrencyModel value.
-	if rs.DeprecatedConcurrencyModel == DeprecatedRevisionRequestConcurrencyModelSingle && rs.ContainerConcurrency == 0 {
-		rs.ContainerConcurrency = 1
+	// is not nil, use the ConcurrencyModel value.
```

Comment:
```suggestion
	// is not (`nil`), use the ConcurrencyModel value.
```

Currently the comment is misleading.